internal/protoplugin/registry.go

```diff
@@ -304,7 +304,11 @@ func defaultGoPackageName(f *descriptor.FileDescriptorProto) string {
 func packageIdentityName(f *descriptor.FileDescriptorProto) string {
 	if f.Options != nil && f.Options.GoPackage != nil {
 		gopkg := f.Options.GetGoPackage()
-		idx := strings.LastIndex(gopkg, "/")
+		idx := strings.Index(gopkg, ";")
```

Comment:
can you please add a comment here? Something like:
`if go_package specifies an alias in the form of full/path/package;alias, use alias over package`

Reply:
Sure, added the comment.